bin/gwdetchar-scattering

```diff
@@ -96,6 +100,12 @@ parser.add_argument('-s', '--segment-start-pad', type=float, default=30.0,
 parser.add_argument('-e', '--segment-end-pad', type=float, default=30.0,
                     help='amount of time to remove from the end of each '
                          'analysis segment')
+parser.add_argument('-lb','--bandpass_low_freq', type=float, default=4.0,
+                     help='the lower limit of the bandpass frequency for'
+                    ' whitened transmons blrms')
+parser.add_argument('-hb','--bandpass_high_freq', type=float, default=10.0,
+                     help='the upper limit of the bandpass frequency for'
+                     'whitened transmons blrms')
```

Comment:
@siddharth101, note that argparse expects single-letter options (which can be uppercase or lowercase), so these two-letter options (i.e. `-lb` and `-hb`) will need to be re-labelled. Also, by convention, the long form of the option uses hyphens instead of underscores, e.g. `--bandpass-low-freq` rather than `--bandpass_low_freq`.